tests/test_core.py

```diff
@@ -49,9 +49,9 @@ def test_invalid_encoding_chunks_with_dask_raise():
     data = dask.array.zeros((10, 20, 30), chunks=expected)
     ds = xr.Dataset({'foo': (['x', 'y', 'z'], data)})
     ds['foo'].encoding['chunks'] = [8, 5, 1]
-    with pytest.raises(ValueError) as excinfo:
```

Comment:
Running against the latest xarray, this throws an `TypeError` because `region` is `None` so it cant even compute the check